porcupine/_state.py

```diff
@@ -10,6 +10,13 @@
 
 from porcupine import images, tabs, utils
 
+try:
+    from ctypes import windll
```

Comment:
This is a bit annoying. Mypy, the type checker that Porcupine uses, complains about this because it doesn't exist on Windows. But if I silence the mypy error by adding a `# type: ignore` comment here, it then complains that the `# type: ignore` comment is unnecessary, which it is on Windows.

The easiest fix is to switch from try/except to `if sys.platform == "win32"`.